test/ctmrg/symmetrization.jl

```diff
@@ -0,0 +1,50 @@
+using Test
+using PEPSKit
+using PEPSKit: herm_depth, herm_width, _fit_spaces
+using TensorKit
+
+function PEPSKit._fit_spaces(data_peps::InfinitePEPS, space_peps::InfinitePEPS)
+    fitted_tensors = map(zip(data_peps.A, space_peps.A)) do (pd, ps)
+        PEPSKit._fit_spaces(pd, ps)
+    end
+    return InfinitePEPS(fitted_tensors)
+end
```

Comment:
we can probably just define this in PEPSKit ? It feels a bit strange to overload a function within the tests